code/controllers/subsystem/job.dm

```diff
@@ -560,6 +569,19 @@ SUBSYSTEM_DEF(job)
 	else if((job.auto_deadmin_role_flags & DEADMIN_POSITION_SILICON) && ((CONFIG_GET(flag/auto_deadmin_silicons) && !timegate_expired) || (C.prefs?.toggles & DEADMIN_POSITION_SILICON))) //in the event there's ever psuedo-silicon roles added, ie synths.
 		return C.holder.auto_deadmin()
 
+/datum/controller/subsystem/job/proc/setup_assistant_positions()
+	var/datum/job/job = SSjob.GetJob("Assistant")
```

Comment:
Don't hardcode "Assistant" anywhere, make this a generically defined variable on `SSjob` much like `overflow_role` is.

Reply:
Or, since the config is named ASSISTANT_SCALING or whatever anyway, a define.